planning/motion_velocity_planner/autoware_motion_velocity_planner_node/src/node.cpp

```diff
@@ -379,8 +379,6 @@ autoware_planning_msgs::msg::Trajectory MotionVelocityPlannerNode::generate_traj
 {
   autoware_planning_msgs::msg::Trajectory output_trajectory_msg;
   output_trajectory_msg.points = {input_trajectory_points.begin(), input_trajectory_points.end()};
-  if (smooth_velocity_before_planning_)
```

Comment:
@maxime-clem This member variable can be removed?

Reply:
It is correctly used in https://github.com/autowarefoundation/autoware.universe/pull/8453
Unless you mean that we should always smooth the velocity before planning, and so we can remove this parameter completely ?